### ISSUE TITLE
Support Legends for > 2 fields

### ISSUE THREAD

**Issue by kanitw** [vega/vega-lite#131]
Also support putting legends on the same side ...

**Comment by kanitw** [comment]
According to https://github.com/vega/vega/issues/140, we can add multiple legends per side in Vega2.

**Comment by kanitw** [comment]
This is automatically done when we use Vega2.   (Checked and found it working in 82d8017cac0dc90cf17bc8b88c32609dcef81f99)

**Comment by kanitw** [comment]
For the recorded, I tested with the following spec:

```
{
  "marktype": "point",
  "encoding": {
    "x": {"name": "Cylinders","type": "O"},
    "y": {"name": "Name","type": "N"},
    "size": {"name": "Acceleration","type": "Q"},
    "color": {
      "scale": {"quantitativeRange": ["#AFC6A3","#09622A"]},
      "name": "Displacement",
      "type": "Q"
    },
    "shape": {"bin": {"maxbins": 15},"name": "Miles_per_Gallon","type": "Q"}
  },
  "data": {"url": "data/cars.json"}
}
```

**Comment by kanitw** [comment]
Actually we should
- [ ] remove comment in legend.js
- [ ] put all legends on one side to save space by default   (maybe `right`?), or lay them out based on size

**Comment by kanitw** [comment]
Fixed in #610 by setting legends’ `orient` to `right` by default (consistent with Vega 2).
